core/management/commands/load_batch.py

```diff
@@ -33,6 +34,9 @@ def add_arguments(self, parser):
         parser.add_argument('--skip-process-ocr', action='store_true',
                             default=True, dest='process_ocr',
                             help='Do not generate ocr, and index')
+        parser.add_argument('--interactive', action='store_true',
```

Comment:
As far as I can tell there's no way to pass `False` to this flag. If you don't set it, it's true by default, and it's a boolean flag, so if it's present it's simply setting interactive to `True`.